lib/phoenix/live_dashboard/system_info.ex

```diff
@@ -231,12 +239,95 @@ defmodule Phoenix.LiveDashboard.SystemInfo do
     Atom.to_string(name) =~ search or String.downcase(desc) =~ search or version =~ search
   end
 
-  defp started_apps_set() do
+  def started_apps_set() do
     Application.started_applications()
     |> Enum.map(fn {name, _, _} -> name end)
     |> MapSet.new()
   end
 
+  def app_tree(app) do
+    master = :application_controller.get_master(app)
+    {child, _app} = :application_master.get_child(master)
+    {children, seen} = sup_tree(child, %{master => true, child => true})
+    {children, _seen} = links_tree(children, master, seen)
+
+    case has_ancestor?(child) do
```

Comment:
Interesting. Which apps would have such ancestors here?

Reply:
There's a node between supervisor and master 
![image](https://user-images.githubusercontent.com/25604253/82764044-b2b2a800-9dd1-11ea-8291-5e5d6a7a5772.png)
And observer draws it 
![image](https://user-images.githubusercontent.com/25604253/82764072-d970de80-9dd1-11ea-939e-90c4fec1bc77.png)

Maybe there's a better solution...

Reply:
Interesting! I will investigate and confirm. :)

Reply:
Yes, this is the correct solution. My only suggestion is to rename `has_ancestor?` to `get_ancestor`, as it doesn't return a boolean. Then make it return `nil` for the negative cases. :)

Reply:
Ok, I renamed the function